extension/src/processExecution.ts

```diff
@@ -26,22 +27,46 @@ export interface ProcessOptions {
   executable: string
   args: string[]
   cwd: string
+  detached?: boolean
   env?: NodeJS.ProcessEnv
 }
 
+const getOptions = (
+  cwd: string,
+  detached?: boolean,
+  env?: NodeJS.ProcessEnv
+) => {
+  const options = {
+    all: true,
+    cwd,
+    env,
+    extendEnv: true,
+    windowsHide: true
+  }
+
+  if (!detached) {
+    return options
+  }
+
+  // a detached/non-shell/windowsHide process will error on Windows:
+  // https://github.com/nodejs/node/issues/21825#issuecomment-503766781
+  // use this workaround: https://github.com/sindresorhus/execa/issues/433
+  const detachedOption =
+    getProcessPlatform() === 'win32' ? { shell: true } : { detached }
```

Comment:
tbh I am not massively confident that this will work as expected on Windows. At least the process doesn't crash.

Reply:
I can test this later today on my Windows machine if that helps.

Reply:
Thanks